data-structures/src/data/structure/stack/overlinkedlist/LinkedStack.java

```diff
@@ -0,0 +1,35 @@
+package data.structure.stack.overlinkedlist;
+
+import data.structure.linkedlist.LinkedList;
+
+public class LinkedStack<E> {
+    private final LinkedList<E> items = new LinkedList<>();

```

Comment:
Love that you did not add a constructor, but used field initialization. Much more concise solution in this case!